windows/src/robocorp/windows/_control_element.py

```diff
@@ -431,11 +431,50 @@ def fmt(s):
     def __repr__(self):
         return f"""{self.__class__.__name__}({self.__str__()})"""
 
+    def _raise_or_warn_control_not_found(
+            self,
+            locator: Locator,
+            search_depth: int,
+            timeout: Optional[float],
+            raise_error: Optional[type],  # to raise or not and with what class
+            exception: Exception,  # previous original raised exception
+    ):
+        from . import config as windows_config
```

Comment:
I think we shouldn't generate the verbose version of the error when `raise_error` is False (as that's slow and this use-case says that the error is expected...).

Reply:
Totally agree, and isn't this already reflected in the current [change](https://github.com/robocorp/robo/pull/100/files#diff-b3a3f67db1c482a1b6c0953faf82ccd4a8d47acdf50c1cf127e8b62d69416c4fR461-R464)? (the long verbose message won't be generated with this set as `None`)

Also, I went with `raise_error` for non-ambiguity (clearer than `default`), but this will take either `None` or the exception type to be raised along with the message.

Reply:
Oh, yes, sorry :)